src/app/pages/msg-sent/msg-sent.component.html

```diff
@@ -1,10 +1,10 @@
 <div class="container vertical-center montserrat">
   <div class="row mx-3 mt-3 justify-content-center">
-    <h1 class="display-4" id="title-text">
+    <h1 class="display-text">
       {{ name ? name : 'No one' }} will receive your message soon!
```

Comment:
Let's not demoralize our users please
```suggestion
      {{ name ? name : 'Your friend' }} will receive your message soon!
```